docs/source/recipes/images.md

```diff
@@ -0,0 +1,51 @@
+# Images
+
+Volto comes with an Image component that allows to load images in an optimal way.
+
+Features of the Image component are:
+
+- optimized lazy loading
+- automatic `srcset` generation, if given a content item or catalog brain
+- optional responsive width
+
+The component can accept all the regular html `<img>` attributes, as well as a few extra for the features described above.
+
+Setting the `loading="lazy"` attribute to the Image component also adds the `decoding="async"` attribute which is generally considered an improvement when loading images lazily.
+
+In order to render an image from a content object, you would use the component like this:
+
+```javascript
+import { Image } from '@plone/volto/components';
+
+<Image item={content} imageField="image" alt="" />;
+```
+
+This will generate the `src`, `srcset`, `width` and `height` attributes automatically for the given image.
+
+If the `responsive` prop is added, a class is applied that will make the image width responsive.
```

Comment:
Switch from passive to active voice, and address the reader.

Also please check these two sentences for accuracy. I cannot tell if the code inserts a provided named CSS class, or the literal `responsive` as the CSS class by default. IOW does it do the following?

`<img class="responsive otherClasses" ...>`

or

`<img class="myCustomNamedResponsiveClass otherClasses" ...>`?

```suggestion
To make the image width responsive, you can add the `responsive` prop with the name of a CSS class, and its name will be applied to the HTML `image` tag's `class` attribute.
If you omit a name, `responsive` will be applied by default.
```

Reply:
I guess it 's a little bit of both. It adds the literal `responsive` as the CSS class, which also has custom CSS rules for making it responsive

Reply:
Got it. What do you think of this version?

```suggestion
To make the image width response, you can add the prop `responsive`, and the literal `responsive` will be added to the HTML attribute `class`.
```

Regarding other HTML attributes, I think an example of how the component would render would help clarify. I'll draft one up above for you to refine.